canvas_modules/common-canvas/src/common-canvas/common-canvas-utils.js

```diff
@@ -616,4 +616,76 @@ export default class CanvasUtils {
 	static isSupernodeBinding(node) {
 		return node.isSupernodeInputBinding || node.isSupernodeOutputBinding;
 	}
+
+	// Returns true if the node passed in is a binding node in a subflow
+	// for a supernode.
+	static isSuperBindingNode(d) {
```

Comment:
How come this function is added? Are we not able to reuse the `isSupernodeBinding(node)` function above?

Reply:
Sorry I meant to tidy this up. There is also an identical method in object-model so I've changed the code there to use the CanvasUtils.isSuperBindingNode(). Thanks! Good catch.